storm-shaded-deps/pom.xml

```diff
@@ -358,5 +357,34 @@
                 </executions>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
+                <plugin>
+                    <groupId>org.eclipse.m2e</groupId>
+                    <artifactId>lifecycle-mapping</artifactId>
+                    <version>1.0.0</version>
```

Comment:
Shall we pull that version up to easier overview for updating?

Reply:
Wanted to keep it localized with the goal of removing this section completely when/if eclipse understands the class relocation. But on second thought, will pull this up.

Reply:
🫠